input/pagecontent/sexgender.md

```diff
@@ -0,0 +1,336 @@
+The guidance below addresses how sex, gender, and related concepts can be structured in FHIR and conformant to AU Base.
+
+### Name to Use (NtU)​
+AU Base supports representation and exchange of the Name to Use data element (as defined in the [HL7 Cross Paradigm Implementation Guide: Gender Harmony - Sex and Gender Representation, Edition 1](https://hl7.org/xprod/ig/uv/gender-harmony/informative1/)) in:
+- [AU Base Patient](StructureDefinition-au-patient.html)
+- [AU Base Practitioner](StructureDefinition-au-practitioner.html)
+- [AU Base Related Person](StructureDefinition-au-relatedperson.html)
+
+Name to Use is represented using "usual" in `name.use`. 
+
+Example: RelatedPerson resource with name to use
+~~~
+{
+  "resourceType" : "RelatedPerson",
+    ...
+    "name" : [
+    {
+      "use" : "usual",
+      "family" : "Franklin",
+      "given" : [
+        "Stella"
+      ]
+    }
+  ],
+  ...
+}  
+~~~
+
+
+### Pronouns​
+AU Base supports representation and exchange of the Pronouns data element (as defined in the [HL7 Cross Paradigm Implementation Guide: Gender Harmony - Sex and Gender Representation, Edition 1](https://hl7.org/xprod/ig/uv/gender-harmony/informative1/)) in:
+- [AU Base Patient](StructureDefinition-au-patient.html)
+- [AU Base Practitioner](StructureDefinition-au-practitioner.html)
+- [AU Base Related Person](StructureDefinition-au-relatedperson.html)
+
+Pronouns is represented with the [Individual Pronouns](http://hl7.org/fhir/StructureDefinition/individual-pronouns) extension. In AU Base the value element of the [Individual Pronouns](http://hl7.org/fhir/StructureDefinition/individual-pronouns) extension is constrained to be [Australian Pronouns](https://www.healthterminologies.gov.au/integration/R4/fhir/ValueSet/australian-pronouns-1) ([extensible](http://hl7.org/fhir/R4/terminologies.html#extensible)).
+
+When populating the value element of the [Individual Pronouns](http://hl7.org/fhir/StructureDefinition/individual-pronouns) extension:
+- *they/them/their/theirs/themselves* may be represented by sending the LOINC Answer (LA) code LA29520-6
+- *she/her/her/hers/herself* may be represented by sending the LOINC Answer (LA) code LA29519-8
+- *he/him/his/his/himself* may be represented by sending the LOINC Answer (LA) code LA29518-0
+- *Prefer not to answer* may be represented by sending the Data Absent Reason code "asked-declined"
+- *Asked but not known* may be represented by sending the Data Absent Reason code "asked-unknown"
+- *Not stated or inadequately described* may be represented by the Data Absent Reason code "unknown"
```

Comment:
Suggest: ... represented by **sending** the ...

Reply:
fixed

Reply:
Confirmed